integrations/server_test.go

```diff
@@ -104,7 +104,8 @@ func TestServer(t *testing.T) {
   "default": true,
   "telegraf": "telegraf",
   "organization": "howdy",
-  "defaultRP": "",
+	"defaultRP": "",
+	"authentication": "basic",
```

Comment:
needs a `gofmt`?

Reply:
These are json strings and this is the result of gofmt